rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -75,6 +76,19 @@
 #define MULTIDOMAIN 0
 #endif
 
+#if RMW_VERSION_GTE(0, 8, 1) && MULTIDOMAIN
+#define SUPPORT_LOCALHOST 1
+#else
+#define SUPPORT_LOCALHOST 0
+#endif
+
+/* QOS Property List support exists in Cyclone if and only if security features are available */
+#if DDS_HAS_SECURITY && DDS_HAS_QOS_PROPERTY_LIST
+#define RMW_SUPPORT_SECURITY 1
+#else
+#define RMW_SUPPORT_SECURITY 0
```

Comment:
I don't believe these lines are necessary. They will [default to 0 in the #if anyway](https://gcc.gnu.org/onlinedocs/gcc-3.0.1/cpp_4.html#SEC39).

Reply:
This follows the cyclone standard--they #define MULITDOMAIN and SUPPORT_LOCALHOST in the same way

Reply:
Fair enough.